packages/itwinui-react/src/core/ColorPicker/ColorPalette.tsx

```diff
@@ -148,8 +150,11 @@ export const ColorPalette = React.forwardRef((props, ref) => {
         {colors &&
           colors.map((_color, index) => {
             const color = getColorValue(_color);
+            const swatchLabel = getColorValue(color).toHslString(true);
```

Comment:
The [NVDA screen reader](https://www.nvaccess.org/download/) reads the `hsla` of `hsla(0, 100%, 50%, 1)` as one word ("his-lea") instead of as four individual letters ("ache-es-el-ea"). Same is the case for `rgba`. But for hex colors, it works alright.

Capitalizing the "hsla"/"rgba" reads it correctly in the screen reader. So I think you should capitalize the string.